src/style-flight-header.scss

```diff
@@ -5,9 +5,11 @@
 
 // Import Auro tokens
 @import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
+@import '../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
```

Comment:
Why is this being added to the scope of this document? Custom properties are to be included at the app global level. This actually causes a scoping issue if the global properties change.

Reply:
Don't need the custom properties, it's been removed.